shared/js/background/storage/config.es6.js

```diff
@@ -0,0 +1,74 @@
+// Babel includes for async/await
+import 'regenerator-runtime/runtime'
+
+const load = require('./../load.es6')
+const constants = require('../../../data/constants')
+const settings = require('./../settings.es6')
+
+/**
+ *  Manage local cookie exclusion data.
+ *  Data is updated regularly with the latest
+ *  cookie exclusions, and stored here for fast retrieval and usage
+ *  when checking which cookies to block.
+ **/
+class ConfigStorage {
+    constructor () {
+        this.config = {}
+    }
+
+    /**
+     * Retrieve the latest lists of excluded cookie domains. Store as needed.
+     */
+    updateConfigData () {
+        console.log('Config: Getting config data')
+        const extensionUrlData = constants.ExtensionConfig
+        const source = extensionUrlData.source || 'external'
+        const configName = extensionUrlData.name
+        const etag = settings.getSetting(`${configName}-etag`) || ''
+        load.loadExtensionFile({ url: extensionUrlData.url, etag: etag, returnType: extensionUrlData.format, source, timeout: 60000 })
```

Comment:
How does this update differ to tds `getLists`? Could we share/reuse some code here?